python/cudf/cudf/utils/ioutils.py

```diff
@@ -93,14 +93,14 @@
 Returns
 -------
 Total number of rows
-Number of row groups
 List of column names
+List of metadata of each row-group
```

Comment:
Can we be explicit that the return type is `List[pyarrow.parquet.RowGroupMetaData]`?

Reply:
This is stale now.